docs/faq.md

```diff
@@ -60,6 +60,21 @@ These are the current penalties that affect both DNS weight and earned FIL:
 - Fraudulent logging (e.g. self-dealing)
 - Multi-noding (Running multiple nodes on the same host)
 
+These are visibile e.g. by hovering over the _Weight_ of a Node on https://dashboard.saturn.tech, or on the _Penalty_ graph of the [Moonlet](https://github.com/31z4/saturn-moonlet).
+
+Here are more details about each kind of penalty, with information what could cause it, and how to remedy if your Node encounters them:
+
+- `error_ratio` is caused by errors as shown in the log of L1 Node container, scuh as:
+  - Node refusing to connect with a client multiple times.
+- `dup_cache_miss_ratio` is caused if a computed error ratio for consecutive duplicate cache misses goes over a threshold.
+- `health_check_failures` are caused by an unexpectedly (not deregistered) unreachable node.
+  - This penalty starts with each health check failure event, and then gradually decreases over ~6h.
+  - Fix this by resolving the root cause of the node unavailability.
```

Comment:
This can be anywhere between Orchestrator (USA), submarine cables, country, DC, router, node. Not always easy to spot